workspaces/desktop/post-install.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
https://www.cyberciti.biz/tips/finding-bash-perl-python-portably-using-env.html

Same below.

Reply:
Resolved in 195ad1dd7c3371ef95516d2dae3e9b13dc2331a0.